dap-etl/extraction/src/main/scala/org/broadinstitute/monster/dap/common/ExtractionPipelineBuilder.scala

```diff
@@ -92,22 +92,23 @@ class ExtractionPipelineBuilder(
     // Group downloaded IDs into batches.
     // NOTE: This logic is replicated from the encode-ingest pipeline,
     // we should consider moving it to scio-utils.
-    val batchedIds = idsToExtract
-    // Pack up the IDs into dummy objects so beam knows how to handle them,
-    // then encode them into UTF8 so they make consistent JSON.
-      .map(KV.of("", _))
-      .setCoder(KvCoder.of(StringUtf8Coder.of(), StringUtf8Coder.of()))
-      // Batch up the IDs
-      .applyKvTransform(GroupIntoBatches.ofSize(idBatchSize.toLong))
-      .map { ids =>
-        // Construct a query to grab the full records for each batch of study IDs
-        GetRecords(
-          ids = ids.getValue.asScala.toList,
-          forms = formsForExtraction,
-          // List of fields to pull out of the data for us to filter on
-          fields = fieldList
-        )
-      }
+
+    val batchedIds: SCollection[GetRecords] =
```

Comment:
👍 iirc from our pairing the only change here is the addition of a type annotation to help the compiler resolve needed implicits.